utils/src/ooinstall/oo_config.py

```diff
@@ -325,6 +326,10 @@ def _set_defaults(self):
         self.settings['ansible_inventory_path'] = \
             '{}/hosts'.format(os.path.dirname(self.config_path))
 
+        # pylint: disable=consider-iterating-dictionary
+        # Disabled because we shouldn't alter the container we're
+        # iterating over
+        #
         # clean up any empty sets
         for setting in self.settings.keys():
             if not self.settings[setting]:
```

Comment:
@tbielawa @smunilla: not related to this PR, but this doesn't seem right to me...  It also looks like we are removing the ansible_ssh_user default we are setting above here, since an empty string is false...  It also seems to exclude the possibility of adding keys with boolean values.